SoftLayer/CLI/hardware/reload.py

```diff
@@ -12,9 +12,9 @@
 
 @click.command()
 @click.argument('identifier')
-@click.option('--postinstall', '-i',
-              help=("Post-install script to download "
-                    "(Only HTTPS executes, HTTP leaves file in /root"))
+@helpers.multi_option('--postinstall', '-i',
```

Comment:
Wait, I don't think you can give multiple post install scripts when reloading hardware. SoftLayer_Container_Hardware_Server_Configuration.customProvisionScriptUri takes in a string, not an array.

ref:
 - http://sldn.softlayer.com/reference/services/SoftLayer_Hardware_Server/reloadOperatingSystem
 - http://sldn.softlayer.com/reference/datatypes/SoftLayer_Container_Hardware_Server_Configuration